faucet/port.py

```diff
@@ -168,7 +176,6 @@ def __init__(self, _id, dp_id, conf=None):
         self.name = None
         self.native_vlan = None
         self.number = None
-        self.op_status_reconf = None
```

Comment:
Did you mean to delete this?

Reply:
Yes that is a vestigial duplicate not referenced anywhere.